src/main/java/com/auth0/net/Request.java

```diff
@@ -19,7 +19,7 @@
      * @throws APIException   if the request was executed but the response wasn't successful.
      * @throws Auth0Exception if the request couldn't be created or executed successfully.
      */
-    T execute() throws Auth0Exception;
+    Response<T> execute() throws Auth0Exception;
```

Comment:
NIT:
`@return the Response object providing the decoded JSON through Response#getBody`

Should we update the doc to something like this?

Reply:
or `{@link Request}` to Request object.

Reply:
👍 good catch, updated the javadoc